pkg/operator/controller/dns_status.go

```diff
@@ -64,7 +65,14 @@ func computeDNSDegradedCondition(oldCondition *operatorv1.OperatorCondition, clu
 		Type: operatorv1.OperatorStatusTypeDegraded,
 	}
 	numberUnavailable := ds.Status.DesiredNumberScheduled - ds.Status.NumberAvailable
+	// TODO: Replace GetValueFromIntOrPercent with
+	// GetScaledValueFromIntOrPercent after rebasing on Kubernetes 1.20.
```

Comment:
These TODO's won't apply to OCP 4.6. Should they be removed, or am I being too picky?

Reply:
Interesting question.  ¯\\_(ツ)\_\/¯

We wouldn't worry about that sort of thing in an automatic cherry-pick, but since this is a manual cherry-pick, we can tailor the change to the release branch.  I'll keep the test as a comment but replace the to-do with an explanation of why it is not enabled on 4.6.  Does that seem reasonable?

Reply:
Maybe I should just copy `GetScaledValueFromIntOrPercent` into `dns_status.go` for the backport.  I'm wishing now I had done that in #217, and then to-do for 4.7 would be just to delete the copy and use the original after the rebase.

Reply:
No worries. Copying over the necessary functions into 4.6 sounds good to me. We can just worry about the rebase in 4.7 now. :grin: